### ISSUE TITLE
WCAG 2.0 failure on buttons

### ISSUE THREAD

**Issue by scotty6435** [jackmoore/colorbox#518]
The replacement of div controls with buttons causes Colorbox to fail WCAG 2.0 validation because they do not have a title or label.  I've tested adding labels successfully, is this something you would be willing to add in?  Because the controls are all image replaced this shouldn't cause any conflicts.

**Comment by lkmorlan** [comment]
Duplicate of #765.
